server/src/triggers.mjs

```diff
@@ -27,6 +27,31 @@ import * as commandLine from './commandLine.mjs';
 const methodTriggers = {};
 const eventTriggers = {};
 
+// Log error
+function logError(errorType, pathDetails, ex) {
```

Comment:
Can we change this name to be more specific to what it's logging?

Reply:
Taken care